mscore/plugin/pluginCreator.cpp

```diff
@@ -90,26 +90,32 @@ PluginCreator::PluginCreator(QWidget* parent)
       actionUndo->setEnabled(false);
       actionRedo->setEnabled(false);
 
+      QToolBar* runTools = addToolBar(tr("Run Operations"));
+      runTools->setObjectName("RunOperations");
+      actionRun->setShortcut(QKeySequence(Qt::Key_F4));
```

Comment:
This is, again, hard-coded. Not good.

Reply:
But QtCreator uses by default Ctrl+R (or Ctrl+B for building); Visual studio switches between Alt+F6 and Ctrl+B; Vs code has Ctrl+Shift+B as a default.
So would Ctrl+B be good for you? As you said, there is no good default. But making it configurable is clearly not worth it. So it will have to be hardcoded.

Reply:
Hardcoding it is clearly wrong, as there is no good setting. It needs to be configurable.

Reply:
Any idea on how it could be made configurable then?

Reply:
What do you mean, clearly wrong? Musescore has hardcoded shortcuts for every single action.
The plugin creator is not an IDE, it does not have to be perfect. Just convenient. So a default shortcut - which might be hardcoded because, as you agreed, there is no standard - needs to be set just for that sake: convenience. 
All we can do is choose something that makes sense, and that's what I am trying to do. If you want that single shortcut to be configurable, I suggest that you open a new PR for it.
Sorry for those tired words, but I don't think that making that single shortcut configurable is worth it.

Reply:
Where do you see hard-coded shortcuts?
See mscore/shortcuts.cpp, for where shortcuts are added (new command "run-plugin"?) and made configurable via Preferences > Shortcuts, and I guess use `addAction(getAction("run-plugin"));` in pluginreator.cpp

Reply:
Well, indeed it seems like a good option! I did not know about this mechanism.
I will try this ASAP.

Reply:
OK, so we talked past one another all the time, it was that mechanism I was referring to from the start ;-)